sdk/go/common/util/deepcopy/copy.go

```diff
@@ -0,0 +1,88 @@
+package deepcopy
```

Comment:
Nit: We probably need to add a copyright header to the top of the file to be consistent with other files in the source tree, right? Same with the new `_test.go` file, too.